site/content/en/docs/manual/advanced/single-shape.md

```diff
@@ -0,0 +1,99 @@
+- [Annotating in Single Shape mode](#annotating-in-single-shape-mode)
+- [Query parameters](#query-parameters)
+- [Video tutorial](#video-tutorial)
+
+## Single Shape mode annotation interface
+
+The interface of the **Single Shape** annotation mode varies based on the selected annotation
+type and the dataset type.
+
+The image below displays the complete interface, featuring all available fields;
+however, certain fields may be absent depending on the scenario. For instance, when annotating
+with rectangles, the **Number of points** field will not appear, and if annotating a single class,
+the **Labels selector** will be omitted.
+
+To access **Single Shape** mode, open the job, navigate to the
+top right corner, and from the drop-down menu, select **Single Shape**.
+
+![Single Shape Annotation Mode Interface](/images/single-shape-interface.png)
+
+The Single Shape annotation mode has the following fields:
+
+<!--lint disable maximum-line-length-->
+
+| Feature                                | Explanation                                                                                                                                                                                                                                                                                                                                                  |
+| -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| **Prompt for Shape and Label**         | Displays the selected shape and label for the annotation task, such as using rectangles to annotate cats.                                                                                                                                                                                                                                                    |
+| **Skip Button**                        | Enables moving to the next frame without annotating the current one, particularly useful when the frame requires no annotation.                                                                                                                                                                                                                              |
+| **List of Hints**                      | Offers guidance on using the interface effectively, including: <br> - Click **Skip** for frames without required annotations. <br> - Hold the **Alt** button to avoid unintentional drawing. <br> - Use the **Ctrl+Z** combination to undo the last annotation if needed. <br> - Use the **Esc** button to completely reset the current annotation progress. |
```

Comment:
```suggestion
| **List of Hints**                      | Offers guidance on using the interface effectively, including: <br> - Click **Skip** for frames without required annotations. <br> - Hold the **Alt** button to avoid unintentional drawing (e.g. when you want only move the image). <br> - Use the **Ctrl+Z** combination to undo the last annotation if needed. <br> - Use the **Esc** button to completely reset the current drawing progress. |
```